README.md

```diff
@@ -1,34 +1,57 @@
 
-Rails Api Base is a boilerplate project for JSON RESTful APIs. It follows the community best practices in terms of standards, security and maintainability, integrating a variety of testing and code quality tools. It's based on Rails 7.0 and Ruby 3.1.
+POC Action Mailbox is a proof of concept of Action Mailbox
+The idea of this development is to learn step by step how Action Mailbox works
 
-Finally, it contains a plug an play Administration console (thanks to [ActiveAdmin](https://github.com/activeadmin/activeadmin)).
+## What is Action Mailbox
+Action Mailbox routes incoming emails to controller-like mailboxes for processing in Rails. 
+Check the ruby doc [here](https://guides.rubyonrails.org/action_mailbox_basics.html). 
 
-## Features
+In other words, the ability to receive emails directly to the ruby app and handle them.
 
-This template comes with:
-- Schema
-  - Users table
-  - Admin users table
-- Endpoints
-  - Sign up with user credentials
-  - Sign in with user credentials
-  - Sign out
-  - Reset password
-  - Get and update user profile
-- Administration panel for users
-- Rspec tests
-- Code quality tools
-- API documentation following https://apiblueprint.org/
-- Docker support
-- Exception Tracking
-- RSpec API Doc Generator
+### Example use case where to use it
 
-## How to use
+imaging we have a client that we need to create a Customer Service Platform, a [mini zendesk's version](https://www.zendesk.com).
+Customer need in bullets:
+1. receive emails from his customer
+2. create a ticket and send a email notification to the customer
+3. Tracking the dialog customer - support. 
+   - The customer or support should replay that email and this action will create a dialog on that ticket
+
+In this POC, we will address the point 1 and 3. 
+
+
+## Working with Action Mailbox in development 
+It's helpful to be able to test incoming emails in development without actually sending and receiving 
+real emails. To accomplish this, there's a conductor controller mounted at
+[rails/conductor/action_mailbox/inbound_emails](localhost:3000/rails/conductor/action_mailbox/inbound_emails), which gives you an index of all the InboundEmails 
+in the system, their state of processing, and a form to create a new InboundEmail as well.
+[doc](https://guides.rubyonrails.org/action_mailbox_basics.html#working-with-action-mailbox-in-development)
+
+**NOTE** 
+To use this feature, you application must not be API only. You can set `config.api_only = false` into config/application.rb file
+
+To use Action Mailbox you must run the following commands:
+
+``` 
+$ bin/rails action_mailbox:install
+$ bin/rails db:migrate 
+```
+
+#### Generate new mailbox
+
+``$ bin/rails generate mailbox forwards``
+
+#### Routing emails
+
+`application_mailbox.rb` file is in charge to root emails using regular expressions. For regular expression, please check 
+[https://rubular.com/](https://rubular.com/)
+
+## How to use this repo
```

Comment:
seems like "repo" is not a valid English word but it's a very common abbreviation on IT for repository so, 🆗